CHANGES.rst

```diff
@@ -15,6 +15,8 @@ Other
 
 - Add wavelength tables for NIRSpec Drizzle cubepars reference file model. [#162]
 
+- Update outlierpars schema to support IFU outlier detection. [#164]
```

Comment:
Should explicitly mention that this is for JWST, because stdatamodels is now a generic package (supposedly). So ... "Update jwst outlierpars schema to support new IFU outlier detection algorithm".